phpBB/phpbb/install/module/update_filesystem/task/file_check.php

```diff
@@ -138,6 +138,15 @@ function ($filename) use ($root_path)
 			$progress_count++;
 			$this->iohandler->set_progress('UPDATE_CHECK_FILES', $progress_count);
 
+			// Do not copy viglink again if the previous version was packaged
+			// with it but it does not exist (e.g. deleted by admin)
+			if (strpos($file, $this->phpbb_root_path . 'ext/phpbb/viglink') !== false &&
+				$this->update_helper->phpbb_version_compare($update_info['version']['from'], '3.2.0', '>=') &&
+				!$this->filesystem->exists($this->phpbb_root_path . 'ext/phpbb/viglink/composer.json'))
+			{
+				continue;
```

Comment:
I missed this part, so it answers to one of my upper questions (about reinstalling viglink if removed) :)